cms/djangoapps/contentstore/api/views/course_quality.py

```diff
@@ -116,25 +106,24 @@ def _execute_method_and_log_time(log_time, func, *args):
             response = dict(
                 is_self_paced=course.self_paced,
             )
-            if not DISABLE_COURSE_CHECKLIST_QUALITY.is_enabled(course_key):
-                if get_bool_param(request, 'sections', all_requested):
-                    response.update(
-                        sections=_execute_method_and_log_time(course_key_harvard, self._sections_quality, course)
-                    )
-                if get_bool_param(request, 'subsections', all_requested):
-                    response.update(
-                        subsections=_execute_method_and_log_time(
-                            course_key_harvard, self._subsections_quality, course, request
-                        )
-                    )
-                if get_bool_param(request, 'units', all_requested):
-                    response.update(
-                        units=_execute_method_and_log_time(course_key_harvard, self._units_quality, course, request)
-                    )
-                if get_bool_param(request, 'videos', all_requested):
-                    response.update(
-                        videos=_execute_method_and_log_time(course_key_harvard, self._videos_quality, course)
+            if get_bool_param(request, 'sections', all_requested):
+                response.update(
+                    sections=_execute_method_and_log_time(course_key_harvard, self._sections_quality, course)
+                )
+            if get_bool_param(request, 'subsections', all_requested):
+                response.update(
+                    subsections=_execute_method_and_log_time(
+                        course_key_harvard, self._subsections_quality, course, request
                     )
+                )
+            if get_bool_param(request, 'units', all_requested):
+                response.update(
+                    units=_execute_method_and_log_time(course_key_harvard, self._units_quality, course, request)
+                )
+            if get_bool_param(request, 'videos', all_requested):
+                response.update(
+                    videos=_execute_method_and_log_time(course_key_harvard, self._videos_quality, course)
```

Comment:
will you remove `_execute_method_and_log_time` in new PR?

Reply:
Yes I intend to remove _execute_method_and_log_time once this PR is merged